Formula/libtecla.rb

```diff
@@ -18,4 +18,29 @@ def install
     system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <locale.h>
+      #include <libtecla.h>
+
+      int main(int argc, char *argv[])
+      {
+        char *line;
+        GetLine *gl;
+
+        setlocale(LC_CTYPE, "");
+
+        gl = new_GetLine(1024, 2048);
+        if(!gl)
+          return 1;
+
+        gl = del_GetLine(gl);
```

Comment:
what value is `gl` assigned to in case of a failure? can you add something similar to lines 35-36 above?